TypeCobol.LanguageServer/TypeCobolLanguageServer.cs

```diff
@@ -278,6 +282,42 @@ public void WholeDocumentChanged(object sender, EventArgs eventArgs)
             UpdateTokensLines(compilationDocument);
         }
 
+        /// <summary>
+        /// LSR Cfg Analyzer ID.
+        /// </summary>
+        public const string lspcfgId = "lsp-cfg";
```

Comment:
`public const` is evil because of compile-time copy of the constant. Use `internal const` instead.

Reply:
to qualify it as evil it's very severe, but it's a lesson for me, ok for internal.